application/src/main/java/dev/ikm/komet/app/App.java

```diff
@@ -769,7 +768,7 @@ private void launchClassicKomet() throws IOException, BackingStoreException {
         }
     }
 
-    private void openImport() {
+    private void openImport(Object destinationTopic) {
```

Comment:
Can we change `Object` type to `FrameworkTopic`?
A comment later in the review suggest we refactor the `LANDING_PAGE_TOPIC`.